src/pipeline.h

```diff
@@ -182,6 +184,8 @@ struct PipelineBaton {
     tiffCompression(VIPS_FOREIGN_TIFF_COMPRESSION_JPEG),
     tiffPredictor(VIPS_FOREIGN_TIFF_PREDICTOR_NONE),
     tiffSquash(false),
+    tiffXres(10),
```

Comment:
Sorry, the URL was truncated, see https://github.com/jcupitt/libvips/blob/d1ca808769339a8856be69b281b72d674a9c0f75/libvips/foreign/tiffsave.c#L305-L306 for the default of 1.